docs/src/main/sphinx/functions/ai.md

```diff
@@ -0,0 +1,136 @@
+# AI functions
+
+## Overview
+
+The AI functions allow you to invoke a large language model (LLM)
+to perform various textual tasks.
+Multiple LLM providers are supported: OpenAI and Ollama.
+
+## Configuration
+
+Because the AI functions require an external LLM service, they are not
+available by default. To enable them, you must configure them using a
+[catalog properties file](catalog-properties), which will register the functions
+under the specified catalog name.
+
+Create a catalog properties file `etc/catalog/ai.properties` that references
+the `ai` connector:
+
+```properties
+connector.name=ai
+```
+
+The AI functions all use the `ai` schema name. To avoid needing to reference
+the functions using their fully qualified name, you can configure the
+`sql.path` [SQL environment property](/admin/properties-sql-environment)
+in the `config.properties` file to include the `ai` catalog and schema:
+
+```properties
+sql.path=ai.ai
+```
+
+## Providers
+
+### OpenAI
+
+The OpenAI provider uses the [OpenAI API](https://platform.openai.com/docs/overview)
+to perform the AI functions:
+
+```properties
+ai.provider=openai
+ai.model=gpt-4o-mini
+ai.openai.api-key=xxx
+```
+
+### Ollama
+
+The Ollama provider uses [Ollama](https://ollama.com/) to perform the AI functions:
+
+```properties
+ai.provider=ollama
+ai.model=llama3.3
+ai.ollama.endpoint=http://localhost:11434
+```
+
+## Functions
+
+:::{function} ai_analyze_sentiment(text) -> varchar
```

Comment:
Pretty nice showcase of the functionality. I appreciate the effort of documenting it. This will likely get traction for this functionality.